src/resources/batches/BatchList.tsx

```diff
@@ -180,6 +180,7 @@ export default function BatchList(): React.ReactElement {
         <FunctionField<Batch>
           label='Reference'
           render={(record) => `${record.vault?.[0]}${record?.batchNumber}`}
+          sortBy='id'
```

Comment:
I'm pretty sure the above should be sorted by `batchNumber`

Reply:
If it is okay for. Can I fix it tomorrow?

Reply:
Completely fine :-D.

Reply:
Thank you. @IanMayo